tola/locale/es/LC_MESSAGES/djangojs.po

```diff
@@ -1629,34 +1630,41 @@ msgstr[1] ""
 "Desagregación guardada y seleccionada automáticamente para todos los "
 "indicadores en %s programas."
 
+#. Translators: Saving to the server succeeded
 #: js/pages/tola_management_pages/country/models.js:215
-#: js/pages/tola_management_pages/country/models.js:227
-#: js/pages/tola_management_pages/program/models.js:194
+#: js/pages/tola_management_pages/organization/models.js:111
+#: js/pages/tola_management_pages/program/models.js:195
+#: js/pages/tola_management_pages/user/models.js:208
 msgid "Successfully saved"
 msgstr "Guardado exitosamente"
 
-#: js/pages/tola_management_pages/country/models.js:233
-#: js/pages/tola_management_pages/program/models.js:198
-msgid "Saving failed"
-msgstr "Falló operación de guardado"
+#. Translators: Saving to the server succeeded
+#: js/pages/tola_management_pages/country/models.js:219
+msgid "Successfully Saved"
+msgstr "Guardado Exitosamente"
+
+#. Translators: Saving to the server failed
+#: js/pages/tola_management_pages/country/models.js:226
+msgid "Saving Failed"
```

Comment:
Should this be a lower case "failed"?  There's a "Saving failed" string further down the file as well.